client/src/components/Login.js

```diff
@@ -21,8 +21,15 @@ export class Login extends Component {
   handleSubmit = event => {
     event.preventDefault();
     const { email, password } = this.state;
+
     getToken(email, password)
-      .then(res => sessionStorage.setItem("token", res.token))
+      .then(res => {
+        console.log(res);
```

Comment:
console.log should always be removed, it should only be used for debugging locally only.

Reply:
Removed